cmd/neofs-cli/modules/container/root.go

```diff
@@ -41,6 +42,7 @@ func init() {
 
 	for _, containerCommand := range containerChildCommand {
 		commonflags.InitAPI(containerCommand)
+		object.InitBearer(containerCommand)
```

Comment:
Why do we need bearer token in container commands? It is applicable only for object operations

Reply:
it has `list-objects` command that calls `ObjectSearch`. dont we need bearer for that?

Reply:
mb it's ok to apply that flag not to every cmd but for `list-objects` only

just did that to continue using cli, did not think much about it

Reply:
Pretty strange to see `list-objects` in `container` command. Anyway, lets add bearer to `list-objects` only.

Reply:
> Pretty strange to see list-objects in container command.

Reply:
> Pretty strange to see list-objects in container command.

Why? do you think that listing objects should be part of `object` cmd? well, MB

> Anyway, lets add bearer to list-objects only.

agree, moved